### DISCUSSION TITLE
Django in prod ! a quick todo of upcoming features

### DISCUSSION THREAD

**Discussion by raphodn** [openfoodfacts/open-prices#421] [Announcements]
Now that #355 is done :rocket: 

## Need to fix

- [x] #407
- [x] #465

## Next/priority issues

Speed up
- [x] #340

Stats
- [x] #419
- [x] #420
- [x] #445
- [x] #463

Onboarding, search, visualizations
- [x] https://github.com/openfoodfacts/open-prices-frontend/issues/364
- [x] https://github.com/openfoodfacts/open-prices-frontend/issues/843
- [x] https://github.com/openfoodfacts/open-prices-frontend/issues/878
- [ ] #414
- [ ] https://github.com/openfoodfacts/open-prices-frontend/issues/302

Price addition
- [x] #317 & #388
- [x] #387
- [ ] #394 & #320
- [x] #396
- [ ] #258

**Comment by raphodn** [comment]
@monsieurtanuki some recent changes : 
- Proof thumbnail image
- User, Location & Product stats (returned with the object)
- Global stats: new endpoint /stats
- The web app is now available at https://prices.openfoodfacts.org (without the /app ; there is a redirection in the meantime)

**Reply by monsieurtanuki** [reply]
Thank you @raphodn, I've just updated https://github.com/openfoodfacts/openfoodfacts-dart/issues/978 accordingly.

**Reply by raphodn** [reply]
Here's a list of recently added fields:
- Location model: `type` (OSM, ONLINE) & `website_url`
- Proof model: `receipt_price_count` & `receipt_price_total`
- Price model: `receipt_quantity`
- Stat model: `location_type_osm_count`, `location_type_online_count`, `proof_type_gdpr_request_count`, `proof_type_shop_import_count`

**Comment by raphodn** [comment]
Will be trying to use more the "Github project" view to keep track of priorities & ongoing work
https://github.com/orgs/openfoodfacts/projects/105/views/2

**Reply by raphodn** [reply]
And created a "Roadmap" issue to track upcoming changes: https://github.com/openfoodfacts/open-prices/issues/541
